atd-vze/src/views/Crashes/Crash.js

```diff
@@ -163,19 +164,21 @@ function Crash(props) {
     geocode_method: geocodeMethod,
     cr3_file_metadata: cr3FileMetadata,
     investigator_narrative_ocr: investigatorNarrative,
-  } = data.atd_txdot_crashes[0];
+  } = !!data?.atd_txdot_crashes[0] ? data?.atd_txdot_crashes[0] : 0;
```

Comment:
If i leave this unhandled we get an error when going to a crash id that doesn't exist because  `data.atd_txdot_crashes[0]` is undefined. So I handled this by setting these values to 0 if the data doesn't exist, which feels kinda weird to do so lmk if there's a better way.

Reply:
I'm not sure if theres a better way to do it, but instead of a 0, what about setting the object to `{}`

Reply:
Since if `data?.atd_txdot_crashes[0]` exists, you get an object and not a number

Reply:
Oo yeah that makes more sense